builtins/amp-img.js

```diff
@@ -167,6 +169,17 @@ export class AmpImg extends BaseElement {
     return true;
   }
 
+  /** @override */
+  firstLayoutCompleted() {
+    const placeholder = this.getPlaceholder();
+    if (isExperimentOn(this.win, 'blurry-placeholder') && placeholder && 
+      placeholder.classList.contains('i-amphtml-blur')) {
+      this.element.classList.add('i-amphtml-fade-out');
```

Comment:
With my comment below, we actually might not need any changes here.

Reply:
So when I take out the overriding of fistLayoutCompleted(), the blur doesn't show up since the togglePlaceholder(false) is hiding it quickly not allowing for the fade out. Is it ok to keep the overriding just to make sure images with blurred placeholders don't call togglePlaceholder(false)?